pinephone/device.mk

```diff
@@ -5,7 +5,7 @@
 $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit.mk)
 $(call inherit-product, device/glodroid/common/device-common.mk)
 
-PRODUCT_PROPERTY_OVERRIDES += ro.sf.lcd_density=269
+PRODUCT_PROPERTY_OVERRIDES += qemu.sf.lcd_density=269
```

Comment:
Are you sure about this? 
Try to drop this line.

Reply:
I'm sure that ro. will makes the DPI set to still 160, which leads to very small UI components.

Reply:
Maybe we need some better way to management the setting of ro.sf.lcd_density .

Reply:
Ah. I see:

```
./frameworks/native/services/surfaceflinger/SurfaceFlinger.cpp:876:                // if "qemu.sf.lcd_density" is specified, it overrides everything
```

Lets keep it as is for now.